@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -312,6 +413,15 @@ export class NullLiteralExpr extends Expr {
     accept<Result, Context>(visitor: ExprVisitor<Result, Context>, context: Context): Result {
         return visitor.visitNullLiteralExpr(this, context);
     }
+
+    /**
+     * Fake `value` getter to be compatible with other literal expressions.
```

Comment:
why do you need this? also `fake value` sounds a bit strange for public API.

Reply:
Each `LiteralExpr` has `value` property. So, null needs to have one too.

Reply:
Is this public API on the other hand, if it's marked with `@hidden` in doc ?

(nevertheless, i've removed offending comment).